configs/Asus/TUF-GAMING-X570-PLUS.conf

```diff
@@ -0,0 +1,88 @@
+chip "nct6798-*"
+# nct6776 values for Asus TUF GAMING X570-PLUS (WI-FI)
+# I'm not entirely sure everything here is correct since no HW monitoring application under Windows shows
+# all the names for all the found sensors. Some of the names and formulas are best guesstimates and could be wrong.
+# You will have to unignore fans and intrusion if you want them to be monitored - I don't have them, I've hidden them
```

Comment:
If you could find out which inputs the remaining fan headers correspond to and comment out/remove the corresponding `ignore` statements here, that would be great. We don't want to hide fans from users.

Also, the lines in this comment are way too long. Wrap at around 80 characters, please.